client/client_webhook.py

```diff
@@ -48,7 +50,24 @@ def __init__(self, commit_data=None, api_url=None, pre_convert_bucket=None, cdn_
         else:
             self.source_url_base = None
 
+        # remove all files from previous process_webhook during testing
+        # files are available after run for audit
+        #tempfile.tempdir = "/tmp/repo"
+        self.intermediate_dir =  'tx-manager'
```

Comment:
Suggest calling this dir `client_webhook`, just in case we also do this for tx-manager, which is actually separate in what it does.